recipes/querystring_parser/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "querystring_parser" %}
+{% set version = "1.2.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4c31b547c77b927a8aceccd6fa37f8152aa92fe5654a43d7363fcee8cf6d8aea
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [not win]
```

Could this be a noarch package instead:
```suggestion
  noarch: python
```